app/src/main/java/com/yeonae/chamelezone/view/course/CourseModifyActivity.kt

```diff
@@ -0,0 +1,262 @@
+package com.yeonae.chamelezone.view.course
+
+import android.Manifest
+import android.net.Uri
+import android.os.Bundle
+import android.os.Handler
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.RelativeLayout
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.gun0912.tedpermission.PermissionListener
+import com.gun0912.tedpermission.TedPermission
+import com.kroegerama.imgpicker.BottomSheetImagePicker
+import com.kroegerama.imgpicker.ButtonType
+import com.kroegerama.kaiteki.toast
+import com.yeonae.chamelezone.Injection
+import com.yeonae.chamelezone.R
+import com.yeonae.chamelezone.data.model.PlaceItem
+import com.yeonae.chamelezone.ext.glideImageSet
+import com.yeonae.chamelezone.ext.shortToast
+import com.yeonae.chamelezone.network.room.entity.UserEntity
+import com.yeonae.chamelezone.view.course.presenter.CourseRegisterContract
+import com.yeonae.chamelezone.view.course.presenter.CourseRegisterPresenter
+import kotlinx.android.synthetic.main.activity_course_register.*
+import kotlinx.android.synthetic.main.slider_item_image.view.*
+
+class CourseModifyActivity : AppCompatActivity(), CourseRegisterContract.View,
+    BottomSheetImagePicker.OnImagesSelectedListener, PlaceCheckDialogFragment.OnClickListener {
+    override fun onClick(place: PlaceItem, placeIndex: Int) {
+        getVisible(place, placeIndex)
+    }
+
+    override lateinit var presenter: CourseRegisterContract.Presenter
+    private var imageUri: String = ""
+    var memberNumber: Int = 0
+    private var firstPlaceNumber: Int = NOT_SELECTED
+    private var secondPlaceNumber: Int = NOT_SELECTED
+    private var thirdPlaceNumber: Int = NOT_SELECTED
+    private val placeNumbers = mutableListOf<Int>()
+    private var isCreated = false
+
+    override fun deliverUserInfo(user: UserEntity) {
+        memberNumber = user.userNumber ?: 0
+    }
+
+    override fun showMessage(message: String) {
+        this.shortToast(message)
+        finish()
+    }
+
+    override fun onImagesSelected(uris: List<Uri>, tag: String?) {
+        toast("$tag")
+        imageContainer.removeAllViews()
+        uris.forEach { uri ->
+            val rl = LayoutInflater.from(this).inflate(
```

Comment:
```suggestion
            val '' = LayoutInflater.from(this).inflate(
```
uri로 이미지를 담을 view를 만드는 작업에 맞는 네이밍으로 수정하면 좋을 것 같아요.

Reply:
수정했습니다
55816fd